projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts

```diff
@@ -147,13 +148,13 @@ context('Tabbing order - tests do require user to be logged in', () => {
 
   describe('Header - Desktop (logged in)', () => {
     it('should allow to navigate with tab key', () => {
-      headerDesktopTabbingOrder(config.headerDesktopLoggedIn);
+      headerTabbingOrder(config.headerDesktopLoggedIn, false, true);
```

Comment:
we should probably think about config parameter - an object with different props, like so:
`{ mobile: true, foo: false }`. it will be much easier for other people to understand what's going on here - instead of a list of true-falses as parameter

Reply:
Yup, we should refactor this into some sort of config object.